identity-apps-core/apps/authentication-portal/src/main/webapp/includes/product-footer.jsp

```diff
@@ -130,7 +130,7 @@
                 List<String> langSwitcherEnabledServlets = Arrays.asList("/oauth2_login.do", "/oauth2_error.do",
                     "/confirmregistration.do", "/confirmrecovery.do", "/claims.do", "/oauth2_consent.do",
                     "/fido2-auth.jsp", "/fido2-identifierfirst.jsp", "/fido2-enroll.jsp", "/fido2-passkey-status.jsp",
-                    "/email_otp.do", "/org_name.do", "/org_discovery.do", "/retry.do", "/totp_enroll.do",
+                    "/fido2-error.jsp", "/email_otp.do", "/org_name.do", "/org_discovery.do", "/retry.do", "/totp_enroll.do",
```

Comment:
Please check whether the content in fido2-error.jsp persists between browser refreshes. If it doesn't, let's not enable lang switcher for this page.

Reply:
The error message persist even after refresh